gix-transport/Cargo.toml

```diff
@@ -78,7 +78,7 @@ base64 = { version = "0.21.0", optional = true }
 curl = { version = "0.4", optional = true }
 
 # for http-client-reqwest
-reqwest = { version = "0.11.12", optional = true, default-features = false, features = ["blocking"] }
+reqwest = { version = "0.11.12", optional = true, default-features = false, features = ["blocking", "rustls-tls"] }
```

Comment:
`rustls-tls` can't be enabled here, it must be up to the user of `gix` to do that.

Doing so seems unrelated to the task at hand, but I see that it's needed to make the feature available in `reqwest`. This also means that in order to ever get there, one would have to wire all reqwest related features though `gix-transport`, which seems like a maintenance nightmare.

Thus, let's remove `reqwest` support for `sslVerify` until there is a solution.

Reply:
I think if this [feature](https://github.com/rust-lang/rust/issues/64797) is ever stabilized, it may solve the issue cleanly.